README.md

```diff
@@ -59,6 +59,18 @@ The avatar attribute on your IdP side may contain a URL pointing to an image fil
 The base64 should start with `data:image/png;base64,` or `data:image/jpeg;base64,`.
 The image should be in JPG or PNG format and have the same width and height.
 
+### Custom login button label
+
+You can set a custom label for the buttons in the login page.
+
+Set this value in `config.php`:
+``` php
+'user_oidc' => [
+    'login_label' => 'Connect with {name}',
```

Comment:
General note currently custom themes would also work to overwrite translations

https://github.com/nextcloud/server/issues/27433 is related as a future enhancement for this